### DISCUSSION TITLE
Error pada Lime

### DISCUSSION THREAD

**Discussion by valmayukuk** [teamalgoritma/community#751] [Q&A] [answered]
Saya mengalami error sebagai berikut

![image](https://github.com/teamalgoritma/community/assets/130435857/49e99511-a50f-4f70-a70a-ae672ca4e14d)

dimana errornya berkata "no applicable method for 'explain' applied to an object of class "factor"

apakah ada solusi terkait ini? karena saya sudah mencocokkan dengan contoh dari https://algoritmaonline.com/text-lime/ tipe tipe datanya untuk dipersiapkan sebelum masuk ke fungsi explain()

**Comment by Yusraf002** [comment]
Dear @valmayukuk ,

Terimakasih atas pertanyaannya bu, tim kami akan segera coba bantu atau carikan referensi untuk solving errornya.

**Comment by arkanayudhistira** [comment]
Dear @valmayukuk,

Berdasarkan code yang ibu gunakan, seharusnya data yang digunakan untuk menjadi input `explain()` adalah text yang ingin di interpretasi (text sms nya dalam bentuk character), bukan statusnya. Silahkan dicoba untuk mengganti nilai dari `text_test` menjadi text sms nya dalam bentuk character.

**Reply by valmayukuk** [reply]
![image](https://github.com/teamalgoritma/community/assets/130435857/f621f0e9-7266-48b6-942b-702db5f60ee6)

Terima kasih kak atas solusinya, namun masih menimbulkan eror baru seperti gambar di atas

**Reply by arkanayudhistira** [reply, marked as answer]
Dear @valmayukuk,

Apakah boleh mengirimkan file Rmd yang ibu gunakan ke email mentor@algorit.ma ? Sehingga kami dapat mencoba untuk mencari penyebab error tersebut.

**Reply by valmayukuk** [reply]
baik mas
